share/pmacc/examples/gameOfLife2D/include/Simulation.hpp

```diff
@@ -92,7 +92,7 @@ namespace gol
 
             /* - This forwards arguments to SubGrid.init()                        *
              * - Create Singletons: EnvironmentController, DataConnector,         *
-             *                      PluginConnector, nvidia::memory::MemoryInfo   */
+             *                      PluginConnector, device::MemoryInfo           */
```

Comment:
I think` #include <pmacc/device/MemoryInfo.hpp>` is (and was) missing.

Reply:
I will remove the comment. Memory info is used over `EnvironementController` therefore there is no direct include required.

Reply:
Ah, sorry, it was at the end of my review and I didn't notice it was a comment.